Final_project/sql/transfer_customers_from_raw_to_bronze.sql

```diff
@@ -0,0 +1,21 @@
+DELETE FROM `{{ params.project_id }}.1_bronze.customers`
+WHERE DATE(_logical_dt) = "{{ ds }}";
+
+MERGE   `{{ params.project_id}}.1_bronze.customers` T
+USING   (SELECT DISTINCT Id, FirstName, LastName, Email, RegistrationDate, State FROM customers_csv) S
+ON  (
+        T.Id = S.Id
+     )
+WHEN matched THEN
+UPDATE
+    SET
+    T.FirstName = IFNULL(T.FirstName, S.FirstName),
```

Comment:
Why do you use IFNULL here?
What if user X changed his/her name? You miss this change if you use IFNULL